renovate.json

```diff
@@ -10,11 +10,7 @@
 			"extends": "monorepo:wordpress",
 			"separateMajorMinor": false,
 			"prPriority": 2,
-			"reviewers": [
-				"@Automattic/cylon",
-				"@Automattic/ganon",
-				"@Automattic/luna"
-			]
+			"reviewers": [ "team:@Automattic/cylon", "team:@Automattic/ganon", "team:@Automattic/luna" ]
```

Comment:
i'm not sure you need the `@` here...

wish there was a way to test / validate this.

Reply:
@rarkins what's the right syntax here? we want to use an organization-level team as a reviewer.  should it be `team:combinedSlug` or `team:slug`? Does the `@` matter?

Reply:
I'm not sure actually. Here's the code: https://github.com/renovatebot/renovate/blob/0b61cfc17109e0dd4fece6d023eefcaa9e265f40/lib/platform/github/index.ts#L1437-L1451

The GitHub docs are not particularly guiding in this matter: https://developer.github.com/v3/pulls/review_requests/#create-a-review-request

Reply:
I would guess you maybe just post `team:slug`. i.e. `team:cylon` on this case.